app/views/support_interface/uploads/new.html.erb

```diff
@@ -1,3 +1,5 @@
+<% content_for :page_title, "Check the Children’s Barred List - Upload records (CSV)" %>
```

Comment:
In `application.html.erb` we do we join on the service name, so these would actually come out as `"Check the Children’s Barred List - Upload records (CSV) - Check the Children’s Barred List"`

```suggestion
<% content_for :page_title, "Upload records (CSV)" %>
```

same for the other page titles below.

Reply:
Ah missed that. Thanks for spotting, will fix